dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -1456,7 +1456,7 @@ async fn cmd_db_sleds(
         Some(filter) => filter,
         None => {
             eprintln!("note: listing all sleds (use -F to filter, e.g. -F in-service)");
```

Comment:
note likely needs to be changed

Reply:
Changed the wording here to note it's "all _commissioned_ sleds". The followon PR to this (#5733) adds a `SledFilter::Decommissioned` that allows us to see the sleds that are left out by default here.